assets/docs/neso613/ASR_TFLite/pre_trained_models/English/1.md

```diff
@@ -0,0 +1,41 @@
+# Lite neso613/ASR_TFLite/pre_trained_models/English/1
+
+TF Lite quantized version of the Conformer to generate text from audio.
+
+<!-- parent-model: neso613/ASR_TFLite/pre_trained_models/1 -->
+<!-- asset-path: legacy -->
```

Comment:
Please replace `legacy` with a URL from which the model can be downloaded. Note that Google Drive links (even if they are public) will not work. [Google Cloud Storage](https://cloud.google.com/storage) or [Firebase Hosting](https://firebase.google.com/docs/hosting) are good alternatives.

Reply:
Google Cloud Storage is free for some time only. So I need to kept model there for always or some time.

Reply:
It only needs to be there until the model is available on tfhub.dev. Once you see your model is available, you can remove it.

Reply:
ok.thanks!